app/src/main/AndroidManifest.xml

```diff
@@ -30,7 +30,7 @@
             android:value="@string/google_maps_key" />
 
```

Comment:
Is Android manifest file supposed to be in the gitignore if it is autobuilt by your idea?

Reply:
No, because i think it should be uniform for all peeps, and the auto-generated gitignore didn't include it, must be for a reason